README.md

```diff
@@ -49,6 +49,7 @@ We rely on different `django` and `mypy` versions:
 
 | django-stubs   | Mypy version | Django version | Django partial support | Python version |
 |----------------|--------------|----------------|------------------------|----------------|
+| 5.1.2          | 1.13.x       | 5.1            | 4.2                    | 3.8 - 3.12     |
```

Comment:
mypy@1.14 also works, to it should be `1.13+` https://github.com/typeddjango/django-stubs/blob/master/requirements.txt#L15

Reply:
We also support 3.13 now